diffs/AaveV3EthereumLido_InterestRateUpdateWETHAndWstETHEthereum_20250721_before_AaveV3EthereumLido_InterestRateUpdateWETHAndWstETHEthereum_20250721_after.md

```diff
@@ -0,0 +1,123 @@
+## Reserve changes
+
+### Reserves altered
+
+#### wstETH ([0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0](https://etherscan.io/address/0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0))
+
+| description | value before | value after |
+| --- | --- | --- |
+| maxVariableBorrowRate | 85.55 % | 40.55 % |
+| variableRateSlope2 | 85 % | 40 % |
```

Comment:
is missing the rf adjustment to 15% https://governance.aave.com/t/risk-stewards-interest-rate-update-weth-and-wsteth-ethereum/22625/2